hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/bigquery/GoogleBigQueryMessageTransformer.java

```diff
@@ -0,0 +1,8 @@
+package pl.allegro.tech.hermes.consumers.consumer.bigquery;
```

Comment:
why this classes is used in a `consumer/bigquery` as it is only used in `consumer/sender/googlebigquery` package ?